### DISCUSSION TITLE
Samsung bootloader is unlockable, regardless their SoC

### DISCUSSION THREAD

**Discussion by ariest-d365bc** [zenfyrdev/bootloader-unlock-wall-of-shame#2] [General]
There's some caveats
1. Any Samsung proprietary security features will breaks
2. Certain basic phone functionality might break as well: https://xdaforums.com/t/a52s-5g-sm-a528b-unlock-bootloader-camera-failed.4336007/
3. Not sure about widevine, it might go lower to L3
4. No well made custom rom to this day is able to use Samsung proprietary VoLTE implementation
5. Knox will be tripped, marking your device as "permanently tampered" by Samsung

**Comment by zenfyrdev** [comment]
Ah, thanks for the info! I did some additional googling, and it seems that North American variants are locked.

**Comment by Eggmanplant** [comment]
Can confirm this. The M51, which is Snapdragon based, is unlockable. I fricking still have L1.
